rpcserver.go

```diff
@@ -372,13 +372,8 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 	}
 
 	if req.Asset.AssetMeta != nil {
-		metaType, err := unmarshalMetaType(req.Asset.AssetMeta.Type)
-		if err != nil {
-			return nil, err
-		}
-
 		seedling.Meta = &proof.MetaReveal{
-			Type: metaType,
+			Type: proof.MetaType(req.Asset.AssetMeta.Type),
```

Comment:
Hmm... I could've sworn I already had this diff in a PR of mine... Or maybe I just forgot one instance of it?